### PR TITLE
Monkey fork support (gym names/defenders/raids)

### PR DESCRIPTION
**Pull request by wateredstorm**
> config/config.php
```
$fork = "monkey";
`````

### CODE REVIEW

File: forks/monkey.php [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      FROM gym_defenders gd
+      LEFT JOIN forts f ON gd.fort_id = f.id
+      WHERE f.external_id = :gymId";
+
+    $gym_defenders = $db->query($query, [":gymId"=>$gymId])->fetchAll(PDO::FETCH_ASSOC);
+
+    $data = array();
+
+    foreach ($gym_defenders as $defender) {
+      $pid = $defender["pokemon_id"];
```
Comment (MonkeyTedBaker) [inline forks/monkey.php]: should unset unnecessary values from $defender

File: forks/monkey.php [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    foreach ($gyms as $gym) {
+      $gym["enabled"] = true;
+      $gym["pokemon"] = [];
+      $gym["guard_pokemon_name"] = i8ln($pokemon_data[$gym["guard_pokemon_id"]]["name"]);
+      $gym["raid_pokemon_name"] = i8ln($pokemon_data[$gym["raid_pokemon_id"]]["name"]);
+      $gym["latitude"] = floatval($gym["latitude"]);
+      $gym["longitude"] = floatval($gym["longitude"]);
+      $gym["last_modified"] = $gym["last_modified"] * 1000;
+      $gym["raid_start"] = $gym["raid_start"] * 1000; // needs to be banged do that raid_start * 1000
+      $gym["raid_end"] = $gym["raid_end"] * 1000; // something has to be changed here so that raid_end is *1000
```
Comment (phonezawphyo) [inline forks/monkey.php]: Remove unnecessary comments. 　　　　　　　　　　　　　　　　　　　👆

File: adapter.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+global $adapter;
```
Comment (MonkeyTedBaker) [inline adapter.php]: This is probably unnecessary 👆

File: forks/monkey.php (hunk 2 above)
Comment (MonkeyTedBaker) [inline forks/monkey.php]: Remove unnecessary comments. 　　　　　　　　　　　　　　　　　　　👆

**Comment by wateredstorm** [comment]
Remember:
- npm run build
- set inside config.php into monkey on fork options

My repo:
https://github.com/wateredstorm/PMSF/tree/monkey?files=1

Also good idea to check:
https://github.com/MonkeyTedBaker/Monocle
Which is the monocle backend for This PR monkey's  wiki is very clean.

**Comment by Glennmen** [comment]
Monkey support has been merged in PR #28

**Pull request closed without merging**